account_invoice_line_description/readme/CONFIGURE.rst

```diff
@@ -0,0 +1 @@
+The user needs the check *Use only product description on invoice lines* setting on the Invoicing Settings.
```

Comment:
This setting needs to be checked on the Access Rights (Technical Settings) of the user

Reply:
Do you mean in alternative?

Reply:
As discussed, there is a global setting in the accounting or you can enable it per user